packages/@aws-cdk/aws-glue-alpha/lib/table.ts

```diff
@@ -181,6 +182,41 @@ export interface TableProps {
    * @default - The parameter is not defined
    */
   readonly enablePartitionFiltering?: boolean;
+
+  /**
+   * The user-supplied properties for the description of the physical storage of this table. These properties help describe the format of the data that is stored within the crawled data sources.
+   *
+   * The key/value pairs that are allowed to be submitted are not limited, however their functionality is not guaranteed.
+   *
+   * Some keys will be auto-populated by glue crawlers, however, you can override them by specifying the key and value in this property.
+   *
+   * @see https://docs.aws.amazon.com/glue/latest/dg/table-properties-crawler.html
+   *
+   * @see https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_EXTERNAL_TABLE.html#r_CREATE_EXTERNAL_TABLE-parameters - under _"TABLE PROPERTIES"_
+   *
+   * @example
+   *
+   *    declare const glueDatabase: glue.IDatabase;
+   *    const table = new glue.Table(this, 'Table', {
+   *      storageParameters: [
+   *          glue.StorageParameter.skipHeaderLineCount(1),
+   *          glue.StorageParameter.compressionType(glue.CompressionType.GZIP),
+   *          glue.StorageParameter.custom('foo', 'bar'), // Will have no effect
+   *          glue.StorageParameter.custom('separatorChar', ','), // Will describe the separator char used in the data
+   *          glue.StorageParameter.custom(glue.StorageParameters.COMPRESSION_TYPE, 'true'), // Will have no effect
```

Comment:
Is this actually possible with the new validation?

Reply:
I've changed it to now be a non-breaking example